1-js/01-getting-started/1-intro/article.md

```diff
@@ -86,14 +86,13 @@ There are at least *three* great things about JavaScript:
 ```compare
 + Full integration with HTML/CSS.
 + Simple things are done simply.
-+ Supported by all major browsers and enabled by default.
++ Support by all major browsers and enabled by default.
```

Comment:
Does not look good.

Reply:
I'm sorry for this long explanation:

The reason I made this change was to help the last line match the structure of the first one. The [changing structure](https://www.iup.edu/writingcenter/writing-resources/grammar/parallel-structure/) of these bullets makes them a little bit harder to follow.

If each bullet starts with a verb, they'll flow more naturally together. A quick test would be to put "Javascript is..." in front of each of these to see if they form a well-structured sentence: 
  + [Javascript is] Fully integrated with HTML/CSS.
  + [Javascript is] Supported by all major browsers and enabled by default.
  + [Javascript is] Able to do simple things in simple ways.

Let me know if you would like me to make these changes or keep as is.